File: Easy/27 - remove-element/Solution.c
int removeElement(int* nums, int n, int val) {
    int d=0;
    for(int i=0;i<n;i++){
        if(nums[i]==val){
            d++;
            for(int j=i;j<n-1;j++){
                nums[j]=nums[j+1];
            }
            continue;
        }
    }
    return n-d+1;
}

File: easy/27 - remove-element/Solution.java
class Solution {
    public int removeElement(int[] a, int v) {
        int n = a.length,j=n-1;
        for(int i=n-1-1;i>=0;i--){
            while(a[j]==v){
                j--;
                if(j==-1)
                return 0;
            }
            if(a[i]==v){
                a[i]^=a[j];
                a[j]=a[i]^a[j];
                a[i]^=a[j];
                j--;
            }
        }
        return j+1;
    }
}